src/resultsdisassemblypage.cpp

```diff
@@ -140,6 +167,13 @@ void ResultsDisassemblyPage::showDisassembly(QString processName, QStringList ar
             QString addrLine = asmTokens.value(0);
             QString costLine = QString();
 
+            if (m_noShowAddress) {
+                QRegExp hexMatcher(QLatin1String("[0-9A-F]+$"), Qt::CaseInsensitive);
```

Comment:
don't use QRegExp in new code, use QRegularExpression instead

pass pattern as QStringLiteral

```
static const auto hexMatcher = QRegularExpression(QStringLiteral("[0-9a-fA-F]+$"));
```